support/windows-server/system-management-components/PowerShell Console characters garbled for CKJ languages

```diff
@@ -0,0 +1,37 @@
+---
+title: PowerShell Console characters garbled for Chinese, Japanese and Korean languages on Windows Server 2022
+description: PowerShell Console characters garbled for Chinese, Japanese and Korean languages on Windows Server 2022.
+ms.date: 08/13/2021
+author: kryalava
+ms.author: kryalava 
+manager: dcscontentpm
+audience: itpro
+ms.topic: troubleshooting
+ms.prod: windows-server
+localization_priority: medium
+ms.reviewer: slee
+ms.prod-support-area-path: PowerShell
+ms.technology: windows-server-system-management-components
+---
+# PowerShell console characters are garbled for Chinese, Japanese and Korean languages on Windows Server 2022
+
+This article provides a resolution to an issue that characters in PowerShell console getting garbled.
+
+_Applies to:_ &nbsp; Windows Server 2022
+_Original KB number:_ &nbsp;
+
+## Symptom
+
+When you launch PowerShell console the characters are getting garbbled. This applies only to Chinese, Japanese and Korean languages SKUs.
+
+## Resolution
+
+To fix the issue, there are 3 workarounds listed below. Either of them can be used.
+
+## Workaround 1: Launch the Powershell from cmd.exe
+
+Start cmd.exe and run powershell.exe from within the cmd console. This opens up a PowerShell console without any font issues.
+
+## Workaround 2: Change the font in the Powershell console
+
+Start PowerShell the normal way. Right click on the Top bar of PowerShell console, select Properties-> Font tab. Goto Font, select `MS Gothic` and click OK. This should resolve the issue.
```

Comment:
Does this only work with `MS Gothic` or would it work with any True Type font?

Does the problem occur with Windows Terminal? If not, that could be another workaround.

Reply:
We tested with `MS Gothic` only and Steve also suggested this. We didn't test with Windows Terminal. I will leave that for now.

Reply:
I believe Windows Terminal has solved this by rendering glyphs from a different font that supports it.  However, I haven't tested this myself.